vortex-filtering/include/vortex_filtering/filters/ipda.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include <vector>
+#include <Eigen/Dense>
+#include <vortex_filtering/filters/pdaf.hpp>
+#include <vortex_filtering/models/dynamic_model_interfaces.hpp>
+#include <vortex_filtering/models/sensor_model_interfaces.hpp>
+
+namespace vortex::filter
+{
+template <concepts::DynamicModelLTVWithDefinedSizes DynModT, concepts::SensorModelLTVWithDefinedSizes SensModT> class IPDA {
+public:
+  static constexpr int N_DIM_x = DynModT::N_DIM_x;
+  static constexpr int N_DIM_z = SensModT::N_DIM_z;
+  static constexpr int N_DIM_u = DynModT::N_DIM_u;
+  static constexpr int N_DIM_v = DynModT::N_DIM_v;
+  static constexpr int N_DIM_w = SensModT::N_DIM_w;
+
+  using T = Types_xzuvw<N_DIM_x, N_DIM_z, N_DIM_u, N_DIM_v, N_DIM_w>;
+
+  using EKF = vortex::filter::EKF_t<N_DIM_x, N_DIM_z, N_DIM_u, N_DIM_v, N_DIM_w>;
+
+  using Gauss_z  = typename T::Gauss_z;
+  using Gauss_x  = typename T::Gauss_x;
+  using Vec_z    = typename T::Vec_z;
+  using GaussMix = vortex::prob::GaussianMixture<N_DIM_x>;
+  using PDAF     = vortex::filter::PDAF<vortex::models::ConstantVelocity, vortex::models::IdentitySensorModel<4, 2>>;
+
+  IPDA() = delete;
+
+  struct Config : public PDAF::Config
+  {
+    double prob_of_survival = 1.0;
+  };
+
+  /// @brief
+  /// @param measurements Measurements to iterate over
+  /// @param probability_of_survival How likely the object is to survive (Ps)
+  /// @param probability_of_detection How likely the object is to be detected (Pd)
+  /// @param clutter_intensity How likely it is to have a false positive
+  /// @param z_pred The predicted measurement
+  /// @return The existence probability
```

Comment:
Change to /** */ style comments as in all the other files
```suggestion
  /** 
   * @brief
   * @param measurements Measurements to iterate over
   * @param probability_of_survival How likely the object is to survive (Ps)
   * @param probability_of_detection How likely the object is to be detected (Pd)
   * @param clutter_intensity How likely it is to have a false positive
   * @param z_pred The predicted measurement
   * @return The existence probability
   */
```